.agents/skills/webawesome/references/components/tab-group.md

```
@@ -0,0 +1,301 @@
+### Slots
+
+| Name | Description |
+| --- | --- |
+| (default) | \`\` Used for grouping tab panels in the tab group. Must be elements. |
+| \`nav\` | \`\` Used for grouping tabs in the tab group. Must be elements. Note that will set this slot on itself automatically. |
+
+### Attributes & Properties
+
+| Name | Description | Reflects |
+| --- | --- | --- |
+| \`activation\` activation | \`'auto' \\| 'manual'\` When set to auto, navigating tabs with the arrow keys will instantly show the corresponding tab panel. When set to manual, the tab will receive focus but will not show until the user presses spacebar or enter. Type Default 'auto' | |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)